### PR TITLE
[production/RRFS.v1] Reorganization of run/nwges directory structure for all remaining RRFS workflow tasks

### PR DESCRIPTION
**Pull request by BenjaminBlake-NOAA**
## DESCRIPTION OF CHANGES: 

- In this PR, the run and nwges directory structure has been updated for all remaining RRFS workflow tasks (not including the ones already modified by PRs #348 , #356 , #358 , and #365 
- The naming convention for ensemble members is changed to m*** (m followed by 3 digits).  So for ensemble member 1, it would be m001.
- Some cleanup of J-jobs, ex-scripts, and associated config file parameters is included with this PR.  Specifically, the removal of the following tasks which are not needed for RRFSv1: ADD_AEROSOL, PROCESS_LIGHTNING, PROCESS_PM, REFL2TTEN, RUN_ENSPOST, RUN_PREPSTART_ENSMEAN, RUN_PREPSTART_FROMEXT
- I apologize that this PR is so large but updating the run/nwges directory structure is a big change that affects many files throughout the workflow.

## TESTS CONDUCTED: 

- The following tasks were successfully tested on WCOSS2 using the DA engineering test (one spinup and one prod cycle): prep_cyc, analysis_gsi_input, analysis_gsi_diag, postanal_input, cldanl_nonvar, run_fcst, save_restart
- I did not test the bufrsnd job, or any of the EnKF / ensemble jobs.  We may need to run a full system test with the production/RRFS.v1 branch in order to make sure those components are working properly with the run/nwges directory structure changes.

### Machines/Platforms:

- WCOSS2
  - [x] Cactus/Dogwood
  - [ ] Acorn
- RDHPCS
  - [ ] Hera
  - [ ] Jet
  - [ ] Orion
  - [ ] Hercules

### Test cases: 

- [ ] Engineering tests
  - [ ] Non-DA engineering test
  - [x] DA engineering test
    - [ ] Retro
    - [ ] Ensemble
    - [x] Parallel
- [ ] RRFS fire weather
- [ ] RRFS_A:
- [ ] RRFS_B:
- [ ] RTMA:
- [ ] Others:

## ISSUE: 

- Resolves issue #237

### CODE REVIEW

**Review by MatthewPyle-NOAA — COMMENTED** [review]
A few initial comments from looking over the changes - nothing crazy.  

Too many changes in some of the scripts to check over carefully by inspection, so will need to see how many issues pop up when doing a test running RRFS with it.  Guessing it will be more than zero, but don't know what they might be right now.

**Comment by MatthewPyle-NOAA** [comment]
@BenjaminBlake-NOAA One more thing came up when trying to do a non-DA engineering test - probably nothing to do with this PR, but I had to remove some FV3GFS type references to get it to build on Hera.  The config keys on GFS_FILE_FMT_* definitions, but non-WCOSS machines are defining FV3GFS_FILE_FMT_* variables.

./non-DA_eng/config.nonDA.grib2.hera.sh:FV3GFS_FILE_FMT_LBCS="grib2"

set_rrfs_config_general.sh:    GFS_FILE_FMT_LBCS="netcdf"

Maybe not critical since this branch is production (WCOSS2) focused, and I only happened to do it on Hera because I already had a session open there.

File: scripts/exrrfs_make_ics.sh
[the remark this answers is not in the document]
Comment (BenjaminBlake-NOAA) [reply scripts/exrrfs_make_ics.sh:1]: Yeah these changes were intentional.  I added them as part of PR #365 but as you noted with NWGES_DIR having the rrfs.${PDY}/${cyc} structure, it is not necessary, so I removed them here.

File: jobs/JRRFS_RUN_POSTANAL
[the remark this answers is not in the document]
Comment (BenjaminBlake-NOAA) [reply jobs/JRRFS_RUN_POSTANAL:1]: That sounds good to me!

**Comment by BenjaminBlake-NOAA** [comment]
@MatthewPyle-NOAA  Even though the production branch is WCOSS2 focused, I think it would be good to be consistent and change FV3GFS to GFS in the other config files too.

File: scripts/exrrfs_nonvarcldanl.sh
[the remark this answers is not in the document]
Comment (BenjaminBlake-NOAA) [reply scripts/exrrfs_nonvarcldanl.sh:1]: Fixed this with my latest commit.

File: jobs/JRRFS_RUN_POSTANAL
[the remark this answers is not in the document]
Comment (BenjaminBlake-NOAA) [reply jobs/JRRFS_RUN_POSTANAL:1]: The ones I found and changed were JRRFS_RUN_GSIDIAG, JRRFS_RUN_ANALYSIS, and JRRFS_RUN_POSTANAL

**Review by MatthewPyle-NOAA — APPROVED** [review]
Thanks for acting on my suggestions - approving.

**Pull request merged**